src/coreclr/nativeaot/Common/src/System/Collections/Generic/LowLevelDictionary.cs

```diff
@@ -20,9 +20,6 @@ namespace System.Collections.Generic
     ** behavior.)
     **
     ===========================================================*/
-#if TYPE_LOADER_IMPLEMENTATION
-    [System.Runtime.CompilerServices.ForceDictionaryLookups]
-#endif
```

Comment:
Does this mean we can start ripping out lazy generics support from the codebase and close underlying bugs like #85184?

Reply:
I have reverted the ForceDictionaryLookupsAttribute change to keep this PR focused.